Source/Core/Core/HW/WiimoteEmu/Extension/Turntable.cpp

```diff
@@ -117,8 +115,8 @@ void Turntable::Update()
 
   // effect dial
   {
-    const ControllerEmu::Triggers::StateData state = m_effect_dial->GetState();
-    const u8 dial = static_cast<u8>(state.data[0] * 0x0F);
+    const auto dial_state = m_effect_dial->GetState();
+    const u8 dial = static_cast<u8>(dial_state.value * 0x0f) + 0x10;
```

Comment:
I'll go ahead and ask: why `0x10` and where does it come from? :)

Reply:
The dial is a 5-bit value. 2^5/2 == 0x10. Did you want some constants? :P

Reply:
Done. :P